core/domain/general_feedback_services.py

```
@@ -288,3 +336,157 @@ def create_platform_report(
+
+    Returns:
+        Tuple of (summaries, next_cursor, more):
+            summaries: List[PlatformFeedbackSummaryDict].
+            next_cursor: Optional[str]. Cursor for the next page, or None.
+            more: bool. True if there are more results beyond this page.
+
+    Raises:
+        ValueError. The dashboard or technical team is invalid.
+    """
+    if dashboard not in feconf.PLATFORM_FEEDBACK_DASHBOARD_CHOICES:
+        raise ValueError('Invalid dashboard: %s' % dashboard)
```

Ruff (0.15.20)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)